### PR TITLE
slow tests by job

### PR DESCRIPTION
**Pull request by clee2000**
separate slow tests by job

not doing test config at the moment because not sure how to handle that slow and default are different configs

### CODE REVIEW

File: .github/workflows/update-slow-tests.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ on:
     - cron: "5 21 * * *"
   # Have the ability to trigger this job manually
   workflow_dispatch:
+  pull_request:
```
Comment (huydhn) [inline .github/workflows/update-slow-tests.yml:9]: This is fine, but I guess you want to gate the uploading part and only print the JSON file when it's a PR

File: torchci/rockset/commons/__sql/slow_tests_by_workflow.sql [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        (
+            select
+                job.head_sha as sha
+            from
+                commons.workflow_job job
+                JOIN commons.workflow_run workflow on workflow.id = job.run_id
+                JOIN push on workflow.head_commit.id = push.head_commit.id
+            where
+                workflow.name = :workflow
+                AND workflow.head_branch LIKE 'master'
```
Comment (huydhn) [inline torchci/rockset/commons/__sql/slow_tests_by_workflow.sql:12]: It's `main` now :)

File: .github/workflows/update-slow-tests.yml (hunk 1 above)
Comment (clee2000) [reply .github/workflows/update-slow-tests.yml]: whoops it was just for testing

**Review by huydhn — APPROVED** [review]

File: torchci/rockset/commons/__sql/slow_tests_by_workflow.sql [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    CONCAT(name, ' (__main__.', classname, ')') as test_name,
+    AVG(time) as avg_duration_sec,
+    base_name,
+FROM
+    duration_per_job
+GROUP BY
+    CONCAT(name, ' (__main__.', classname, ')'),
+    base_name
+HAVING
+    AVG(time) > 60.0
```
Comment (huydhn) [inline torchci/rockset/commons/__sql/slow_tests_by_workflow.sql:99]: Nit: You could have this 60s as a threshold parameter set by `torchci/scripts/updateSlowTests.mjs`

**Pull request closed without merging**